src/universal/modules/meeting/components/AvatarGroup/AvatarGroup.js

```diff
@@ -16,7 +16,7 @@ const AvatarGroup = (props) => {
       {
         avatars.map((avatar, index) =>
           <div className={css(styles.item)} key={index}>
-            <Avatar {...avatar} size="small" hasBadge hasBorder={avatar.isFacilitator} />
+            <Avatar {...avatar} size="small" hasBadge hasBorder={avatar.isFacilitating} />
```

Comment:
isFacilitator = has the privilege attached to the teamMember document.
isFacilitating = the active facilitator